elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -377,5 +379,22 @@ $pfe-cta--Color--fallback: #003366;
     --pfe-cta--Color--hover:            #{pfe-color(ui-complement)};
 }
 
+/// ===========================================================================
+/// WIND CTA VARIANTS
+/// ===========================================================================
 
-
+:host([pfe-priority="secondary"][pfe-variant="wind"]) {
+  --pfe-cta--BackgroundColor:           transparent;
+  --pfe-cta--BorderColor:               var(--pfe-theme--surface--border, #d2d2d2);
+  --pfe-cta--Color:                     var(--pfe-broadcasted--color--ui-link, #{pfe-color(ui-link)});
+  font-weight:                          #{pfe-var(font-weight--normal)};
```

Comment:
Need to add this as a variable since this should only be updating variables and not setting properties directly.  Add the property to the host and update the value here.